themes/plumerai-blog/layouts/partials/header.html

```diff
@@ -1,5 +1,17 @@
+<a id="home"></a>
 <header>
-    <div class="centered-column">
-        <h1><a href="/">Plumerai Blog</a></h1>
+    <a href="/">
+        <img id="logo-blog" src="/images/logo_wordmark_blog.svg" alt="Plumerai blog logo" />
+        <img id="logo" src="/images/logo_wordmark.svg" alt="Plumerai logo" />
```

Comment:
Actually this seems to make anti aliasing a bit better.
```suggestion
        <img id="logo-blog" src="/images/logo_wordmark_blog.svg" alt="Plumerai blog logo" />
```

Reply:
Yeah, unfortunately the two images are a solution for something else 😆 Will try and find another way of doing it.